editor/sidebar/last-revision/index.js

```diff
@@ -81,6 +81,10 @@ class LastRevision extends Component {
 		const { revisions } = this.state;
 		const lastRevision = revisions.length ? revisions[ 0 ] : null;
 
+		if ( revisions.length <= 0 ) {
+			return false;
```

Comment:
switched this to `return null` as this is in fact what the React docs recommend, see https://facebook.github.io/react/docs/conditional-rendering.html